src/main/java/me/partlysanestudios/partlysaneskies/CustomMainMenu.java

```diff
@@ -127,15 +129,15 @@ public void populateGui(float scaleFactor) {
         String image;
 
         if (PartlySaneSkies.config.customMainMenuImage == 0) {
-            image = "textures/gui/main_menu/" + imageIdMap.get(Utils.randInt(1, imageIdMap.size()));
+            image = "textures/gui/main_menu/" + imageIdMap.get(MathUtils.INSTANCE.randInt(1, imageIdMap.size()));
```

Comment:
Do you really need that INSTANCE everywhere

Reply:
If you're writing in java yes

However in Kolin it's even better

```kt
10.36381.round(2)
```

This is what we had to do before

```kt
Utils.round(10.36381, 2)
```

Reply:
So to remove it we would need everything in kotlin

Reply:
Yeah but I think what we should do is just do everything in the future in Kotlin, and eventually we'll rewrite the old

Reply:
Agree